src/examples/index.tsx

```diff
@@ -6,13 +6,17 @@ import websocket from './websocket-gemini.yml';
 import mqtt from './streetlights-mqtt.yml';
 import simple from './simple.yml';
 
+// tutorial example
+import tutorial from './tutorials/tutorial.yml';
```

Comment:
In the future we will have more tutorials so it's better to call it as (e.g.) `invalid` to describe purpose of the tutorial.

```suggestion
// tutorial example
import invalid from './tutorials/invalid.yml';
```